src/main/java/org/springframework/samples/petclinic/system/CacheConfiguration.java

```diff
@@ -19,8 +19,10 @@ public JCacheManagerCustomizer petclinicCacheConfigurationCustomizer() {
     }
 
     private javax.cache.configuration.Configuration<Object, Object> cacheConfiguration() {
-        // Create a cache using infinite heap. A real application will want to use a more fine-grained configuration,
-        // possibly using an implementation-dependent API
+        // Enable statistics via the JCache programmatic configuration API.
+        // The actual default size limit of the cache is implementation dependent and cannot be modified
```

Comment:
I am not sure that's a valid alternative now you've removed the previous content. The intent of the original description was to mention that this was configuring a cache in-memory. 

I agree the "infinite heap" part is irrelevant. At the time I meant to focus on the fact that using EhCache's specific API allows you to restrict the size in heap and various other options. 

Can you please rephrase the description to focus on the original intent (that this configuration does not configure any sort of persistence for the cache).